src/azure-cli/azure/cli/command_modules/appconfig/_help.py

```diff
@@ -121,6 +121,16 @@
     text: az appconfig kv export -n MyAppConfiguration -d appconfig --dest-name AnotherAppConfiguration --key * --label * --dest-label ExportedKeys
 """
 
+helps['appconfig kv export-secret'] = """
+type: command
+short-summary: Export values of key vault reference from your App Configuration.
```

Comment:
>values [](start = 22, length = 6)

The "values"  being exported may be confused with values of app config keys. Can we make it something like "Export Key Vault secret values from Key Vault references in your App Configuration"?